pkg/recommendation/recommender/resource/recommend.go

```diff
@@ -131,6 +131,13 @@ func (rr *ResourceRecommender) Recommend(ctx *framework.RecommendationContext) e
 		memQuantity := resource.NewQuantity(v, resource.BinarySI)
 		cr.Target[corev1.ResourceMemory] = memQuantity.String()
 
+		//use memory-to-vCPU-ratio if exist
+		if rr.MemTovCPURatioBool {
+			cpu, mem := rr.getVMSpec(cpuQuantity, memQuantity)
```

Comment:
if configuration exist and pass the format checking, then call rr.getVMSpec.
Also we should add a log here to know it happens.